generator/csv/fields.csv

```diff
@@ -377,133 +377,133 @@ State,sp_change_max,f,Int32,0x41,0,0,0,Integer
+Terms,shop_purchased3,f,DBString,0x4A,'',1,0,String
+Terms,shop_sell_select3,f,DBString,0x4B,'',1,0,String
+Terms,shop_sell_number3,f,DBString,0x4C,'',1,0,String
+Terms,shop_sold3,f,DBString,0x4D,'',1,0,String
+Terms,inn_a_greeting_1,f,DBString,0x50,'',1,0,String
+Terms,inn_a_greeting_2,f,DBString,0x51,'',1,0,String
+Terms,inn_a_greeting_3,f,DBString,0x52,'',1,0,String
+Terms,inn_a_accept,f,DBString,0x53,'',1,0,String
+Terms,inn_a_cancel,f,DBString,0x54,'',1,0,String
+Terms,inn_b_greeting_1,f,DBString,0x55,'',1,0,String
+Terms,inn_b_greeting_2,f,DBString,0x56,'',1,0,String
+Terms,inn_b_greeting_3,f,DBString,0x57,'',1,0,String
+Terms,inn_b_accept,f,DBString,0x58,'',1,0,String
+Terms,inn_b_cancel,f,DBString,0x59,'',1,0,String
+Terms,possessed_items,f,DBString,0x5C,'',1,0,String
+Terms,equipped_items,f,DBString,0x5D,'',1,0,String
+Terms,gold,f,DBString,0x5F,'',1,0,String
+Terms,battle_fight,f,DBString,0x65,'',1,0,String
+Terms,battle_auto,f,DBString,0x66,'',1,0,String
+Terms,battle_escape,f,DBString,0x67,'',1,0,String
+Terms,command_attack,f,DBString,0x68,'',1,0,String
+Terms,command_defend,f,DBString,0x69,'',1,0,String
+Terms,command_item,f,DBString,0x6A,'',1,0,String
+Terms,command_skill,f,DBString,0x6B,'',1,0,String
+Terms,menu_equipment,f,DBString,0x6C,'',1,0,String
+Terms,menu_save,f,DBString,0x6E,'',1,0,String
+Terms,menu_quit,f,DBString,0x70,'',1,0,String
+Terms,new_game,f,DBString,0x72,'',1,0,String
+Terms,load_game,f,DBString,0x73,'',1,0,String
+Terms,exit_game,f,DBString,0x75,'',1,0,String
+Terms,status,f,DBString,0x76,'',1,1,String
+Terms,row,f,DBString,0x77,'',1,1,String
+Terms,order,f,DBString,0x78,'',1,1,String
+Terms,wait_on,f,DBString,0x79,'',1,1,String
+Terms,wait_off,f,DBString,0x7A,'',1,1,String
+Terms,level,f,DBString,0x7B,'',1,0,String
+Terms,health_points,f,DBString,0x7C,'',1,0,String
+Terms,spirit_points,f,DBString,0x7D,'',1,0,String
+Terms,normal_status,f,DBString,0x7E,'',1,0,String
+Terms,exp_short,f,DBString,0x7F,'',1,0,String - char x 2?
+Terms,lvl_short,f,DBString,0x80,'',1,0,String - char x 2?
+Terms,hp_short,f,DBString,0x81,'',1,0,String - char x 2?
+Terms,sp_short,f,DBString,0x82,'',1,0,String - char x 2?
+Terms,sp_cost,f,DBString,0x83,'',1,0,String
+Terms,attack,f,DBString,0x84,'',1,0,String
+Terms,defense,f,DBString,0x85,'',1,0,String
+Terms,spirit,f,DBString,0x86,'',1,0,String
+Terms,agility,f,DBString,0x87,'',1,0,String
+Terms,weapon,f,DBString,0x88,'',1,0,String
+Terms,shield,f,DBString,0x89,'',1,0,String
+Terms,armor,f,DBString,0x8A,'',1,0,String
+Terms,helmet,f,DBString,0x8B,'',1,0,String
+Terms,accessory,f,DBString,0x8C,'',1,0,String
+Terms,save_game_message,f,DBString,0x92,'',1,0,String
+Terms,load_game_message,f,DBString,0x93,'',1,0,String
+Terms,file,f,DBString,0x94,'',1,0,String
+Terms,exit_game_message,f,DBString,0x97,'',1,0,String
+Terms,yes,f,DBString,0x98,'',1,0,String
+Terms,no,f,DBString,0x99,'',1,0,String
 Music,name,f,String,0x01,"(OFF)",1,0,String
```

Comment:
Are music name and sound name the only non-save chunks shared with savegames?

Reply:
I believe so, and maybe we'll just convert all the save data too at some point